cli/azd/pkg/project/service_target_appservice.go

```diff
@@ -13,24 +13,27 @@ import (
 	"github.com/azure/azure-dev/cli/azd/pkg/azure"
 	"github.com/azure/azure-dev/cli/azd/pkg/environment"
 	"github.com/azure/azure-dev/cli/azd/pkg/infra"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
 	"github.com/azure/azure-dev/cli/azd/pkg/tools"
 	"github.com/azure/azure-dev/cli/azd/pkg/tools/azcli"
 )
 
 type appServiceTarget struct {
-	env *environment.Environment
-	cli azcli.AzCli
+	env     *environment.Environment
+	cli     azcli.AzCli
+	console input.Console
```

Comment:
Do we still need to add `console` here? I'm not seeing it used in the latest change.